src/App.jsx

```diff
@@ -0,0 +1,19 @@
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import ProductList from "./components/ProductList/productList";
+import { Cart } from "./components/cart/cartList";
+import { WishList } from "./components/WishList/wishlist";
+// import ProductList from "./components/Products";
+import { DataLoader } from "./components/DataLoader";
```

Comment:
- Inside the components folder make an `index.js` file which re-exports all files inside components, so similarly create such file for nested folders you create
```suggestion
import ProductList from "./components";
```